libtracepoint-control-cpp/include/tracepoint/TracepointSession.h

```diff
@@ -829,7 +915,7 @@ namespace tracepoint_control
         _Success_(return == 0) int
         SavePerfDataFile(
             _In_z_ char const* perfDataFileName,
-            int mode = -1) noexcept;
+            TracepointSavePerfDataFileOptions const& options = TracepointSavePerfDataFileOptions()) noexcept;
```

Comment:
Note: Change from taking the mode as an optional parameter to taking an options object as an optional parameter. Not a breaking change because we haven't made any release with the SavePerfDataFile method yet.